http-servers/html-server.js

```diff
@@ -0,0 +1,18 @@
+const http = require("http");
+const fs = require("fs");
+
+http.createServer()
+	.on("request", (req, res) => {
+		res.setHeader("Content-Type", "text/html");
+
+		try {
+			const data = fs.readFileSync("./index.html");
+
+			res.statusCode = 200;
+			res.end(data.toString().replace("{message}", "Hello world!"));
```

Comment:
Please consider using `res.write()` instead of `res.end()` everywhere, because `res.end()` stands for closing the responce process